src/main/kotlin/com/openlattice/codex/CodexService.kt

```diff
@@ -252,9 +275,37 @@ class CodexService(
         return EntityDataKey(entitySetId, entityKeyId)
     }
 
+    fun getScheduledMessagesForOrganization(organizationId: UUID): Map<UUID, MessageRequest> {
+        return BasePostgresIterable(PreparedStatementHolderSupplier(hds, GET_SCHEDULED_MESSAGES_FOR_ORG_SQL) {
+            it.setString(1, organizationId.toString())
+        }) {
+            val task = ResultSetAdapters.schedulableTask(it).task as SendCodexMessageTask
```

Comment:
This is now `scheduledTask` in your c-c PR.

Reply:
ooh yes thanks!!